tools/SetupFlow/DevHome.SetupFlow/ViewModels/AppManagementViewModel.cs

```diff
@@ -67,7 +66,7 @@ public partial class AppManagementViewModel : SetupPageViewModelBase
     protected async override Task OnEachNavigateToAsync()
     {
         SelectDefaultView();
-        await _packageCatalogListViewModel.LoadCatalogsAsync();
+        await Task.CompletedTask;

```

Comment:
Nit: maybe a comment here why you're including this (ie. the other OnEachNavigateToAsyncs do awaits, or this will be replaced with code that needs to be awaits).

Reply:
Can you please elaborate on this comment 🙂 Is the suggestion about removing the `LoadCatalogsAsync` from here or adding `Task.CompletedTask`?

Reply:
I admittedly had to look up what Task.CompletedTask does so maybe my comment is too influenced by what I read... basically I think it might be helpful to explain why Task.CompletedTask is in there.

Reply:
Gotcha! This method is designed to run async, however since there is no tasks to await for inside of it, it's generally a common practice to add an `await Task.CompletedTask` to maintain the async signature. If this method was not `override`, then I would have changed the signature to be sync instead. Note that CS1998 warning will prevent successfully building the project if there was no await inside an async, because it just means the method should have been sync instead (which generally is a good hint, but does not apply in this case of `override`).